.github/workflows/test-publish-dry-run.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
           java-version: ${{ matrix.java }}
 
       - name: Set Gradle Version
-        run: ./gradlew wrapper --gradle-version $VERSION_GRADLE --distribution-type=all
+        run: ./gradlew wrapper --gradle-version=$VERSION_GRADLE --distribution-type=all
```

Comment:
for some reason getting:

> A problem occurred evaluating project ':examples:android-gradle'.
> > Failed to apply plugin 'com.android.internal.version-check'.
>    > Minimum supported Gradle version is 7.3-rc-1. Current version is 7.2. If using the gradle wrapper, try editing the distributionUrl in /home/runner/work/sentry-android-gradle-plugin/sentry-android-gradle-plugin/gradle/wrapper/gradle-wrapper.properties to gradle-7.3-rc-1-all.zip

Reply:
Because AGP 7.2 requires Gradle 7.3 (to run any command, even `gradlew wrapper`). That's a hard requirement